set-integration-testing/src/integration-test/java/eu/europeana/api/set/integration/client/BaseUserSetClientTest.java

```diff
@@ -8,6 +8,8 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Properties;
+
+import eu.europeana.api.commons_sb3.auth.service.GrantConstants;
```

Comment:
we should not use underscore in package names, either commonssb3 or commons3 or better commons.sb3 , but indeed this is refactoring for api-commons-sb3

Reply:
still, I would prefer to copy the code to api-commons-v2, or otherwise to move the set-client to own repository